README.md

```diff
@@ -33,25 +21,26 @@ Binaries and dependency information for Maven, Ivy, Gradle and others can be fou
 
 <a href='http://search.maven.org/#search%7Cga%7C1%7Crxandroid'><img src='http://img.shields.io/maven-central/v/io.reactivex/rxandroid.svg'></a>
 
-Example for Maven:
+Example for Gradle:
+
+```groovy
+compile 'io.reactivex:rxandroid:1.0.0'
+```
+
+and for Maven:
```

Comment:
Do we even need to include the Maven/Ivy versions? I'm thinking we could remove them for brevity. Not many people are using either these days and I think it can be assumed that if you are you know how to convert from gradle syntax to Maven/Ivy syntax.